.github/workflows/label_when_reviewed_workflow_run.yml

```
@@ -97,21 +97,23 @@ jobs:
           require_committers_approval: 'true'
           pullRequestNumber: ${{ steps.source-run-info.outputs.pullRequestNumber }}
           comment: >
-            The PR needs to run all tests because it modifies core of Airflow! Please rebase it to latest
-            master or ask committer to re-run it!
+            The PR most likely needs to run full matrix of tests because it modifies parts of the core
+            of Airflow. However, committers might decide to merge it quickly and take the risk.
+            If they don't merge it quickly - please rebase it to latest master at your convenience,
+            or amend the last commit of the PR, and push it with --force.
```

```suggestion
            or amend the last commit of the PR, and push it with `--force-with-lease`.
```

Safer than `--force`

---

Good point! Changed everyewhere.